tools/ebpf/dev_setup.sh

```diff
@@ -6,9 +6,9 @@ cd $cwd
 set -e
 
 if [[ -f Vagrantfile ]]; then
-  echo "detected vagrant file; will clean up"
-  vagrant destroy -f
-  rm Vagrantfile
+    echo "detected vagrant file; will clean up"
+    VAGRANT_CWD=$(pwd) vagrant destroy -f
```

Comment:
What is this?

Reply:
This is just to avoid destroying the default environment for vagrant if it's enabled.

I have a `VAGRANT_CWD` path set so I can run `vagrant` in any directory and it will always use the same Vagrantfile except when I change the `VAGRANT_CWD` variable.